config.json.example

```diff
@@ -25,5 +25,7 @@
 	"end_of_day": "00:00",
 	"full_team_names": true,
 	"scrolling_speed": 2,
-	"debug": false
+	"debug": false,
+	"demo": false,
+	"demo_date": "2017-07-07"
```

Comment:
I think we should condense this down to one `demo_date` option and have it default to `false`. Then the code can just check if the flag is truthy and parse the date if it's filled out.